<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>New_outfit</class>
 <widget class="QDialog" name="New_outfit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>838</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SuperPuperNew</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #FFFAE6;</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>811</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="styleSheet">
       <string notr="true">font: 800 italic 16pt &quot;Ubuntu&quot;;</string>
      </property>
      <property name="title">
       <string/>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="styleSheet">
                <string notr="true">color: #002379;</string>
               </property>
               <property name="text">
                <string>Type</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="typeEdit">
               <property name="styleSheet">
                <string notr="true"> background-color: #FF9F66;
            border-radius: 100px;
            color: green;
            font-family: CerebriSans-Regular,-apple-system,system-ui,Roboto,sans-serif;
            padding: 7px 20px;
            text-align: center;
            font-size: 16px;
border-radius: 16px;
            border-width: 0 0 4px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="styleSheet">
                <string notr="true">color: #002379;</string>
               </property>
               <property name="text">
                <string>Color</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="colorEdit">
               <property name="styleSheet">
                <string notr="true">background-color: #FF9F66;
			 opacity: 0.3;
            border-radius: 100px;
            color: green;
            font-family: CerebriSans-Regular,-apple-system,system-ui,Roboto,sans-serif;
            padding: 7px 20px;
            text-align: center;
            font-size: 16px;
border-radius: 16px;
            border-width: 0 0 4px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="styleSheet">
                <string notr="true">color: #002379;
</string>
               </property>
               <property name="text">
                <string>Material</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="materialEdit">
               <property name="styleSheet">
                <string notr="true">background-color: #FF9F66;
            border-radius: 100px;
            color: green;
            font-family: CerebriSans-Regular,-apple-system,system-ui,Roboto,sans-serif;
            padding: 7px 20px;
            text-align: center;
            font-size: 16px;
border-radius: 16px;
            border-width: 0 0 4px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="styleSheet">
                <string notr="true">color: #002379;</string>
               </property>
               <property name="text">
                <string>Style</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="styleEdit">
               <property name="styleSheet">
                <string notr="true">background-color: #FF9F66;
            border-radius: 100px;
            color: green;
            font-family: CerebriSans-Regular,-apple-system,system-ui,Roboto,sans-serif;
            padding: 7px 20px;
            text-align: center;
            font-size: 16px;
border-radius: 16px;
            border-width: 0 0 4px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="styleSheet">
                <string notr="true">color: #002379;</string>
               </property>
               <property name="text">
                <string>Season</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="seasonEdit">
               <property name="styleSheet">
                <string notr="true">background-color: #FF9F66;
            border-radius: 100px;
            color: green;
            font-family: CerebriSans-Regular,-apple-system,system-ui,Roboto,sans-serif;
            padding: 7px 20px;
            text-align: center;
            font-size: 16px;
border-radius: 16px;
            border-width: 0 0 4px;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="styleSheet">
              <string notr="true">background-color: #FF5F00;
            border: solid transparent;
            border-radius: 16px;
            border-width: 0 0 4px;
            color: #FFFFFF;
            font-family: din-round,sans-serif;
            font-size: 15px;
            font-weight: 700;
            letter-spacing: .8px;
            line-height: 20px;
            margin: 0;
            outline: none;
            padding: 13px 16px;
            text-align: center;
            text-transform: uppercase;
            vertical-align: middle;
            white-space: nowrap;
            width: 100%;</string>
             </property>
             <property name="text">
              <string>Add picture</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="styleSheet">
            <string notr="true">background-color: #FF5F00;
            border: solid transparent;
            border-radius: 16px;
            border-width: 0 0 4px;
            color: #FFFFFF;
            font-family: din-round,sans-serif;
            font-size: 15px;
            font-weight: 700;
            letter-spacing: .8px;
            line-height: 20px;
            margin: 0;
            outline: none;
            padding: 13px 16px;
            text-align: center;
            text-transform: uppercase;
            vertical-align: middle;
            white-space: nowrap;
            width: 100%;</string>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="view"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
